packages/@aws-cdk/aws-lambda-nodejs/lib/builder.ts

```diff
@@ -111,11 +111,11 @@ export class Builder {
         '-v', `${this.options.projectRoot}:${containerProjectRoot}`,
         '-v', `${path.resolve(this.options.outDir)}:${containerOutDir}`,
         ...(this.options.cacheDir ? ['-v', `${path.resolve(this.options.cacheDir)}:${containerCacheDir}`] : []),
-        '-w', path.dirname(containerEntryPath),
+        '-w', path.dirname(containerEntryPath).replace(/\\/g, '/'), // Always use POSIX paths in the container
```

Comment:
sounds like something we should take to the bundling PR, no?

Reply:
In #7898 we explicitly ask for a `workingDirectory` (optional prop) so the user knows that it's a POSIX style path?

Here we had to calculate it using `path` logic because we are mounting `projectRoot` but want to work closer to the entry file.